sections/staking/components/StakingInfo/InfoLayout.tsx

```diff
@@ -120,16 +118,14 @@ const InfoLayout: FC<InfoLayoutProps> = ({ stakingInfo, collateral, isInputEmpty
 							<RowValue>
 								{formatCurrency(currencyKey, value.toString(), {
 									currencyKey: currencyKey,
-									decimals: 2,
 								})}
 							</RowValue>
 							{!isInputEmpty && (
 								<>
 									<StyledArrowRight src={ArrowRightIcon} />
 									<RowValue>
-										{formatCurrency(currencyKey, !changedValue.isNaN() ? changedValue : 0, {
+										{formatCurrency(currencyKey, !changedValue ? changedValue : 0, {
```

Comment:
I don't think this is correct.

`changedValue || 0`

Reply:
fixed something here so marking resolved